pkg/v1/tkg/test/framework/schemes.go

```diff
@@ -55,4 +56,7 @@ func AddDefaultSchemes(scheme *runtime.Scheme) {
 
 	// Add the run v1alpha3 scheme
 	_ = runv1.AddToScheme(scheme)
+
+	// Add the run v1alpha3 scheme
```

Comment:
nit: This comment doesn't make sense, it should be kapp controller v1alpha?

Reply:
Thanks for catching this. updated